flink-python/pyflink/table/tests/test_udf.py

```diff
@@ -96,6 +96,89 @@ def test_udf_in_join_condition(self):
         actual = source_sink_utils.results()
         self.assert_equals(actual, ["2,Hi,2,Flink"])
 
+    def test_udf_with_constant_param(self):
```

Comment:
1. test_udf_with_constant_param -> test_udf_with_constant_params
2. move it after test_udf_in_join_condition_2?